### ISSUE TITLE
CLI Enhancement: Ability to specify all command parameters in a json file ("command-params-file" flag)

### ISSUE THREAD

**Issue by kcq** [slimtoolkit/slim#597]
Right now each CLI command accepts a set of command parameters as separate command line flags. It's ok for adhoc use, but it's less ideal for automation. The goal for the "command-params-file" flag is to provide an ability to specify all required command-specific parameters in one JSON formatted file.

**Comment by MeenuyD** [comment]
Hello @kcq I would like to work on this issue can you please assign it to me and tell me which files need to be changed

**Comment by thecoons** [comment]
Can I inquire whether the need is to place the parameter values in a new configuration file or if they should be included within "slim.config.json"? 

Because its role seems very similar to the requested feature, but it is limited to a parameter list from [GlobalAppOptions](https://github.com/slimtoolkit/slim/blob/d1107958b636ac4eeac3b0c2dc533b70bfc62681/pkg/app/master/config/config.go#L26).

Do you want to separate the application configurations from the command configurations into two different files? 

(This implies that the values in this new file can override the values in slim.config.json, which adds complexity to the CLI's behavior.)

**Comment by vash-knives** [comment]
@kcq , still need this done? let me know, can work on this.

**Comment by dpcamargo** [comment]
@kcq is this issue still open?

**Comment by kcq** [comment]
> @kcq is this issue still open?

yes, it is open. if you are interested in working on it it's good to post the PRs [here](https://github.com/mintoolkit/mint/pulls)

**Comment by Billy-North** [comment]
I was  just been playing around with this issue yesterday and think I have a solution outlined, I would be keen to try and post a PR over this weekend.
